src/components/Calculator.css

```diff
@@ -0,0 +1,74 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-size: 100%;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calculator {
+  margin: 100px;
+  width: 45%;
+}
+
+.screen input {
+  width: 100%;
+  height: 60px;
+  background: #b5b5bd;
+  border: 1px solid #8f8f8f;
+  border: 0;
+  color: #fff;
+  padding: 10px;
+  text-align: right;
+  font-size: 24px;
+}
+
+.row {
+  font-size: 24px;
+}
+
+.first-row,
+.second-row,
+.third-row,
+.fourth-row,
+.fifth-row {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+}
+
```

Comment:
At screen widths around 400px, the image is disorganized hence does not match the given design as shown below :point_down: . Kindly make sure that it looks like the actual design in all screen widths to make your project more proffessional and up to standard.  


### Your design around 400px (width)

![Screenshot from 2023-03-21 11-33-42](https://user-images.githubusercontent.com/65233860/226597450-4e3c7e5a-e246-44f8-a078-d7185676aacb.png)


### Actual design 


![Screenshot from 2023-03-21 11-50-26](https://user-images.githubusercontent.com/65233860/226597465-da6fd523-ca98-46fe-9b32-3d5fe7290446.png)